doc/safekeep.backup.txt

```diff
@@ -190,6 +190,14 @@ are separated by '/', attributes are introduced by '@':
        This is planned to be specific to the current rdiff-backup, and
        different options will be made available for other backends.
 
+/backup/options/rdiff-backup/@exec_local::
+       Executable to be run on the local host.
+       Optional, defaults to 'rdiff-backup'
+
+/backup/options/rdiff-backup/@exec_remote::
+       Executable to be run on the remote host.
+       Optional, defaults to 'rdiff-backup'
+
```

Comment:
how are these options used? Would be good if an example of backup options could added in the examples?

Reply:
Yes, that is how it is used, although I'll see if I can add an example within the file.

Reply:
would be nice if an example could be added before release as the right syntax is crucial to have correct in order to not start the backup with the wrong version.